src/nvim/eval/funcs.c

```diff
@@ -6027,10 +6028,15 @@ static void get_maparg(typval_T *argvars, typval_T *rettv, int exact)
       } else {
         rettv->vval.v_string = (char_u *)str2special_save((char *)rhs, false, false);
       }
+    } else if (rhs_lua != LUA_NOREF) {
+      size_t msglen = 100;
+      char *msg = (char *)xmalloc(msglen);
+      snprintf(msg, msglen, "<Lua function %d>", mp->m_luaref);
```

Comment:
should we use desc here if available? Maybe `<Lua function: %s>` with the desc instead of opaque int.

Reply:
I have thought having same representation like `:map` command will be better . Also makes it consistent with when desc isn't given . the desc is provided when `dict` option is used on `maparg`

Reply:
> I have thought having same representation like :map command will be better . 

I agree, and shouldn't `:map` use desc as well? also seems more helpful than showing opaque number to the user.

Reply:
`:map` shows desc in 2nd line . Because non lua keymaps can also have desc I feel like it'll be weird if for lua desc apeared in place of rhs and for non lua it apeared somewhere else.

Currently it gets shown like this.
```
:map xx
lhs   flags   rhs
                    desc
```

Reply:
ok, let's keep it like it is for now. we can always develop it further until 0.7 anyway.